### PR TITLE
Fix/news details responsive

### PR DESCRIPTION
**Pull request by wafakd**
Full Activity News on mobile

### CODE REVIEW

File: webapp/src/main/webapp/css/news.less [hunk 1]
```diff
@@ -224,6 +225,14 @@
     }
   }
 }
+@media screen and (max-width: 769px) {
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/css/news.less]: Most of the time in the project platform-ui we use 767 or 768. Do you know where this value comes from ?

Comment (wafakd) [reply webapp/src/main/webapp/css/news.less]: yes, because ipad viewer width is 768px

Comment (thomasdelhomenie) [reply webapp/src/main/webapp/css/news.less]: So we should put 768 no ?

Comment (wafakd) [reply webapp/src/main/webapp/css/news.less]: yes we can put exactly 768px, ok

**Pull request merged**